code/lib/cli/src/automigrate/fixes/autodocs-true.ts

```diff
@@ -51,10 +92,11 @@ export const autodocsTrue: Fix<AutodocsTrueFrameworkRunOptions> = {
     `;
   },
 
-  async run({ result: { main }, dryRun }) {
+  async run({ result: { main, value }, dryRun }) {
     logger.info(`✅ Setting 'docs.autodocs' to true in main.js`);
     if (!dryRun) {
-      main.setFieldValue(['docs', 'autodocs'], true);
+      main.removeField(['docs', 'docsPage']);
+      main.setFieldValue(['docs', 'autodocs'], value ?? true);
```

Comment:
If value is undefined do we want it to default to `true` or to `'tag'`?

Reply:
To `true`. That means they didn't previously have a value, which means they are upgrading, which means we want to set the behaviour to completely automatic.